src/Whiteboard_14_MaxStack/max.java

```diff
@@ -0,0 +1,31 @@
+package Whiteboard_14_MaxStack;
+
+import java.util.Stack;
+
```

Comment:
This new class should be called `MaxStack` - `max` will be the name of the method.